invenio_base/utils.py

```diff
@@ -2,16 +2,44 @@
 #
 # This file is part of Invenio.
 # Copyright (C) 2017-2020 CERN.
+# Copyright (C) 2025 Graz University of Technology.
 #
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 """Base utilities."""
 
+import importlib.metadata as m
+from collections.abc import Callable
+from sys import version_info
+from typing import Union
+
 from flask import current_app
 from werkzeug.utils import import_string
 
 
+def entry_points(
+    entry_point_group: str,
+    func: Callable = None,
+    sorted_options: Union[dict, None] = None,
+) -> Union[set[m.EntryPoint], None]:
+    """Entry points."""
+    if version_info < (3, 10):
+        eps = m.entry_points().get(entry_point_group, [])
+    else:
+        eps = m.entry_points(group=entry_point_group)
```

Comment:
minor/comment: I see the distinction between Python versions, but I think the type of `eps` is tricky between versions:

- [in Python 3.9 it's a `dict`](https://docs.python.org/3.9/library/importlib.metadata.html#entry-points)
- [in Python 3.10+ it's a "collection" type named `EntryPoints`](https://docs.python.org/3.10/library/importlib.metadata.html#entry-points). In later versions it also has new attributes to help with filtering/grouping

What result does running `sorted` and `set` have on these variants?